lib/Minz/Url.php

```diff
@@ -94,27 +96,23 @@ private static function printUri(array $url, string $encodage): string {
 			unset($url['params']['c']);
 			unset($url['params']['a']);
 			foreach ($url['params'] as $key => $param) {
-				if (!is_string($key) || (!is_string($param) && !is_int($param))) {
+				if (!is_string($key) || (!is_string($param) && !is_int($param) && !is_bool($param))) {
 					continue;
 				}
 				$uri .= $separator . urlencode($key) . '=' . urlencode((string)$param);
 				$separator = $and;
 			}
 		}
 
-		if (!empty($url['#']) && is_string($url['#'])) {
```

Comment:
@math-GH If you can, please check that this has not caused any regression since https://github.com/FreshRSS/FreshRSS/pull/4416